docusaurus/docs/protocol/primitives/claim_and_proof_lifecycle.md

```diff
@@ -19,13 +19,11 @@ to all readers.
   - [Session Duration](#session-duration)
```

Comment:
I'm noticing that the adding a table of contents to the contents seems redundant as docusaurus builds one in the sidebar on the right.

![image](https://github.com/pokt-network/poktroll/assets/600733/036d8128-da8d-4844-acdd-eaedc400583a)

Reply:
I still think it's useful when rendering locally.